packaging/bundle-validation/README.md

```diff
@@ -39,17 +39,17 @@ the folder. Here are the docker commands to build the image by specifying differ
 ```shell
 docker build \
  --build-arg HIVE_VERSION=3.1.3 \
- --build-arg FLINK_VERSION=1.14.6 \
+ --build-arg FLINK_VERSION=1.15.3 \
  --build-arg SPARK_VERSION=3.1.3 \
  --build-arg SPARK_HADOOP_VERSION=2.7 \
- -t hudi-ci-bundle-validation-base:flink1146hive313spark313 .
-docker image tag hudi-ci-bundle-validation-base:flink1146hive313spark313 apachehudi/hudi-ci-bundle-validation-base:flink1146hive313spark313
+ -t hudi-ci-bundle-validation-base:flink1153hive313spark313 .
```

Comment:
we only have `flink1153hive313spark331` `flink1153hive313spark332` `flink1153hive313spark334`